core/src/ast_builder/expr/function.rs

```diff
@@ -44,16 +55,20 @@ pub fn abs<T: Into<ExprNode>>(expr: T) -> ExprNode {
     ExprNode::Function(Box::new(FunctionNode::Abs(expr.into())))
 }
 
+pub fn ifnull<T: Into<ExprNode>, V: Into<ExprNode>>(expr: T, then: V) -> ExprNode {
+    ExprNode::Function(Box::new(FunctionNode::IfNull(expr.into(), then.into())))
+}
+
 pub fn floor<T: Into<ExprNode>>(expr: T) -> ExprNode {
     ExprNode::Function(Box::new(FunctionNode::Floor(expr.into())))
 }
 
 #[cfg(test)]
 mod tests {
-    use crate::ast_builder::{abs, col, expr, floor, test_expr};
+    use crate::ast_builder::{abs, col, expr, floor, test_expr, text};
 
     #[test]
-    fn function_abs() {
+    fn abs_test() {
```

Comment:
Since we used the `#[test]` macro, it turns out that this is a test function.
It seems good to omit the test prefix.

example)
https://github.com/gluesql/gluesql/blob/main/core/src/ast_builder/select/having.rs#L73-L74
https://github.com/gluesql/gluesql/blob/main/core/src/plan/evaluable.rs#L175-L176

Reply:
I agree. I really wanted to just write it like `abs`, `ifnull`, `floor`.

But I got an `error[E0255]: the name abs is defined multiple times` message when I do so (because of line 68).

I'll rollback the changes to `function_abs` format for now.